// ngx-fs/__ivy_ngcc__/index.js
import { NgModule } from '@angular/core';
import { FsService } from './src/fsService';
import { FsServiceRef } from './src/fsServiceRef';
import * as ɵngcc0 from '@angular/core';
export * from './src/fsService';
export * from './src/fsServiceRef';
var NgxFsModule = (function () {
    function NgxFsModule() {
    }
NgxFsModule.ɵfac = function NgxFsModule_Factory(t) { return new (t || NgxFsModule)(); };
NgxFsModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: NgxFsModule });
NgxFsModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ providers: [{ provide: FsService, useClass: FsServiceRef }] });
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(NgxFsModule, [{
        type: NgModule,
        args: [{
                declarations: [],
                exports: [],
                providers: [{ provide: FsService, useClass: FsServiceRef }]
            }]
    }], function () { return []; }, null); })();
    return NgxFsModule;
}());
export { NgxFsModule };
/** @nocollapse */
NgxFsModule.ctorParameters = function () { return []; };
